yarn.lock

```diff
@@ -10844,7 +10825,7 @@ core-js@^2.4.0, core-js@^2.5.0, core-js@^2.6.9:
   resolved "https://registry.yarnpkg.com/core-js/-/core-js-2.6.9.tgz#6b4b214620c834152e179323727fc19741b084f2"
   integrity sha512-HOpZf6eXmnl7la+cUdMnLvUxKNqLUzJvgIziQ0DiF3JwSImNphIqdGqzj6hIKyX04MmV0poclQ7+wjWvxQyR2A==
 
-core-js@^3.0.4, core-js@^3.21.1, core-js@^3.6.5, core-js@^3.8.2, core-js@^3.8.3:
```

Comment:
@elastic/kibana-operations can you please check this?
it looks like that the `core-js@^3.8.3` was already in the `yarn.lock` and running `yarn kbn bootstrap` just forced the use of the standalone one. Should I revert this change?

Similar thing happened with `canvg` and `@type/raf` that where listed 2 times in the yarn lock

Reply:
+1 on the revert, this one doesn't make sense to me.  3.8.3 -> 3.8.2 with no changes in dependencies in charts, I'm not sure how it happened